boards.txt

```diff
@@ -25,26 +25,26 @@ generic.serial.disableRTS=true
 
 generic.build.mcu=esp8266
 generic.build.f_cpu=80000000L
-generic.build.f_crystal=1
 generic.build.board=ESP8266_ESP01
 generic.build.core=esp8266
 generic.build.variant=generic
 generic.build.flash_mode=qio
 generic.build.spiffs_pagesize=256
 generic.build.debug_port=
 generic.build.debug_level=
+generic.build.extra_flags=-DF_CRYSTAL={build.f_crystal}
```

Comment:
I suppose that Travis build is failing because I'm using an older Arduino IDE version there?

Anyway, I'm even okay with having `f_crystal=26000000` specified for every board. Hope this can fix the build issue.